cmd/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/sungatuly22/tournament-service/server"
+)
+
+func main() {
+	srv := server.NewServer()
+	srv.Routes()
+	http.ListenAndServe(":8080", srv)
```

Comment:
Uncomment server.ListenAndServe and use here instead of http.ListenAndServer.
P.S. do not forget about the returning error